lib/Doctrine/ORM/EntityRepository.php

```diff
@@ -132,7 +132,7 @@ public function createNativeNamedQuery($queryName)
      */
     public function clear()
     {
-        $this->_em->clear($this->_class->rootEntityName);
+        $this->_em->clear();
```

Comment:
I would rather deprecated `EntityRepository::clear()`

Reply:
That's what #7922 suggests anyway, and I added it to the list of issues your PR closes because I think you should go with that solution.

Reply:
Exactly, removing the argument is also a bc break since it will clear everything.